perllib/FixMyStreet/Integrations/ExorRDI.pm

```diff
@@ -116,13 +119,15 @@ sub construct {
                 $location .= " Nearest postcode: $closest_address->{postcode}{postcode}." if $closest_address->{postcode};
             }
 
-            my $description = sprintf("%s %s", $report->external_id || "", $report->get_extra_metadata('detailed_information') || "");
+            my $traffic_information = $report->get_extra_metadata('traffic_information') || 'none';
+            my $description = sprintf("%s %s %s %s",
+                $report->external_id || "",
+                $initials,
+                'TM' . ($tm_lookup{$traffic_information} || '0'),
+                $report->get_extra_metadata('detailed_information') || "");
```

Comment:
There might be a few inspections done before this is deployed that have a 172-length `detailed_information`, which should probably be handled here.